superset/db_engine_specs/ocient.py

```diff
@@ -224,16 +224,21 @@ class OcientEngineSpec(BaseEngineSpec):
         "P0.25Y": "ROUND({col}, 'QUARTER')",
         "P1Y": "ROUND({col}, 'YEAR')",
     }
+    
 
     @classmethod
     def get_table_names(
         cls, database: Database, inspector: Inspector, schema: Optional[str]
     ) -> List[str]:
         return sorted(inspector.get_table_names(schema))
+    
 
     @classmethod
     def fetch_data(cls, cursor, lim=None):
-        rows = super(OcientEngineSpec, cls).fetch_data(cursor)
+        try:
+            rows = super(OcientEngineSpec, cls).fetch_data(cursor)
+        except Exception:
+            rows = []
```

Comment:
we swallow the exception from the db????

Reply:
Fixed, error is now propagated